docs/userGuide/contentAuthoring.md

```diff
@@ -435,14 +435,15 @@ If you would like to have a site navigation bar for your site, you may create on
     - Ensure you wrap your navigation layout in a `<navigation>` tag for MarkBind to render it properly. Also, there should only be one `<navigation>` tag in your file.
     - You may author additional HTML and Markdown content outside the `<navigation>` tag.
     - The `<navigation>` tag is also optional, allowing you to author everything in HTML and Markdown syntax.
+    - Use `:expanded:` keyword at the end of Dropdown titles to expand them by default. 

```

Comment:
This is not strictly true. The current algorithm allows adding `:expanded:` at **any** location in the Dropdown title, not just at the end only.

Reply:
Maybe create a test case for one that does not have `:expanded:` at the back? All of them currently have the keyword at the back at least.